cmd/flux/create_helmrelease.go

```diff
@@ -239,6 +241,9 @@ func createHelmReleaseCmdRun(cmd *cobra.Command, args []string) error {
 		for _, v := range helmReleaseArgs.valuesFiles {
 			data, err := os.ReadFile(v)
 			if err != nil {
+				if helmReleaseArgs.ignoreMissingValueFiles && os.IsNotExist(err) {
+					continue
+				}
```

Comment:
This is not what the flag should, we should map it to HelmRelease spec instead of skipping over value files passed to the CLI.

Reply:
I think PR is probably unnecessary then. It only made sense to me if you were passing `--values-files` and `--ignore-missing-values-files` together (which we're not ourselves planning to do). Let's drop it.